.bazelrc

```diff
@@ -6,6 +6,7 @@ build:engflow_common --experimental_inmemory_dotd_files
 build:engflow_common --experimental_inmemory_jdeps_files
 build:engflow_common --incompatible_strict_action_env=true
 build:engflow_common --remote_timeout=3600
+build:engflow_common --enable_platform_specific_config
```

Comment:
```suggestion
build --enable_platform_specific_config
```

Reply:
To explain: each line starts with a verb like `build`, meaning it applies to `build` commands or anything inherited from a `build` command like `test` or `run`. If a verb has a suffix like `:engflow_common`, the flags are not enabled by default, but they can be used by running Bazel with `--config=engflow_common`. Multiple `--config` flags are allowed.

The configurations `engflow` and `engflow_oss` below "inherit" from `engflow_common` by using `--config=engflow_common`.

The flag `--enable_platform_specific_config` flag tells Bazel to act as if `--config=linux` or `--config=macos` were passed on the command line (whichever one matches the host platform). We want those flags to be on generally, not only when we are using remote execution.